lib/test.js

```diff
@@ -58,8 +58,16 @@ function configure(sinon, config) {
             } catch (e) {
                 exception = e;
             }
-
-            if (typeof oldDone !== "function") {
```

Comment:
I just meant a little line to help late comers reading the code. Some might forget that then is an attribute of Promise and wonder what this does. The code seems fine.